OpenLIFUProtocolConfig/OpenLIFUProtocolConfig.py

```diff
@@ -108,6 +110,67 @@ class OpenLIFUProtocolConfigParameterNode:
 
     """
 
+#
+# OpenLIFUProtocolConfigDialogs
+#
+
+class ProtocolSelectionFromDatabaseDialog(qt.QDialog):
+    """ Create new protocol selection from database dialog """
+
+    def __init__(self, protocol_names_and_IDs : "List[Tuple[str,str]]", parent=None):
+        super().__init__(slicer.util.mainWindow() if parent == "mainWindow" else parent)
+        """ Args:
+                dataLogic: instance of the OpenLIFUData module logic, needed to access the loaded database
+        """
+
+        self.setWindowTitle("Select a Protocol")
+        self.setWindowModality(qt.Qt.WindowModal)
+        self.resize(600, 400)
+
+        self.protocol_names_and_IDs : "List[Tuple[str,str]]" = protocol_names_and_IDs
```

Comment:
```suggestion
        self.protocol_names_and_IDs : List[Tuple[str,str]] = protocol_names_and_IDs
```

(we generally don't need quotes on type annotation, we only need those for the lazy importing stuff)